sbt-plugin/src/main/scala/scala/scalajs/sbtplugin/ScalaJSPluginInternal.scala

```diff
@@ -401,6 +427,46 @@ object ScalaJSPluginInternal {
               "are running a JVM REPL. JavaScript things won't work.")
       )),
 
+      scalaJSNativeLibraries := {
+        collectFromClasspath(fullClasspath.value,
+            "*.js", collectJar = _.jsFiles,
+            collectFile = FileVirtualJSFile.relative)
+      },
+
+      jsDependencyManifests in jsManifestFilter := {
+        collectFromClasspath(fullClasspath.value,
+            new ExactFilter(JSDependencyManifest.ManifestFileName),
+            collectJar = _.jsDependencyManifests,
+            collectFile = { (file, _) =>
+              fromJSON[JSDependencyManifest](readJSON(IO.read(file)))
+            })
+      },
+
+      jsDependencyManifests := {
+        val filter = jsManifestFilter.value
+        val rawManifests = (jsDependencyManifests in jsManifestFilter).value
```

Comment:
In fact, `jsManifestFilter` could even be deprecated, suggesting to replace
```scala
jsManifestFilter ~= { _.andThen(f) }
```
by
```scala
jsDependenciesManifests ~= { _.map(f) }
```
Couldn't it?